__tests__/app.test.js

```diff
@@ -29,7 +30,43 @@ describe('/api', () => {
             .get('/api')
             .expect(200)
             .then((response) => {
-                expect(response.body).toEqual(endpoints);
+                expect(response.body.endpoints).toEqual(endpoints);
             });
     });
+});
+
+describe('/api/articles/:article_id', () => {
+    test('GET:200 responds with an article object ', () => {
+        return request(app)
+            .get('/api/articles/5')
+            .expect(200)
+            .then((response) => {
+                expect(response.body.article).toEqual({
```

Comment:
Due to the fact that this endpoint is going to change later on in the project, we should use a more flexible matcher here. Take a look at toMatchObject or objectContaining which will allow you to do this. 👍